fastlane/lib/fastlane/actions/delete_keychain.rb

```diff
@@ -4,25 +4,17 @@ module Fastlane
   module Actions
     class DeleteKeychainAction < Action
       def self.run(params)
-        original = Actions.lane_context[Actions::SharedValues::ORIGINAL_DEFAULT_KEYCHAIN]
-
-        search_paths = []
-        search_paths << File.expand_path(params[:name]) if params[:name]
-        search_paths << File.expand_path(File.join("~", "Library", "Keychains", params[:name])) if params[:name]
-        search_paths << File.expand_path(params[:keychain_path]) if params[:keychain_path]
```

Comment:
Before, this would look for all the possible names of the keychain. Could you not add another search path with the '-db' filename? The `search_paths.find` will be sure to find the one path